File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/ImpressaoCodigoBarras.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImpressaoCodigoBarras(
    private val tipo: Int,
    private val dados: String,
    private val altura: Int,
    private val largura: Int,
    private val HRI: Int
) : TermicaCommand("ImpressaoCodigoBarras") {
    override fun functionParameters(): String {
        return "\"tipo\"" + ":" + tipo + "," +
                "\"dados\"" + ":" + "\"" + dados + "\"" + "," +
                "\"altura\"" + ":" + altura + "," +
                "\"largura\"" + ":" + largura + "," +
                "\"HRI\"" + ":" + HRI
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/impressao_codigo_barras.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class ImpressaoCodigoBarras extends TermicaCommand {
  final int _tipo;
  final String _dados;
  final int _altura;
  final int _largura;
  // ignore: non_constant_identifier_names
  final int _HRI;

  ImpressaoCodigoBarras(
      this._tipo, this._dados, this._altura, this._largura, this._HRI)
      : super('ImpressaoCodigoBarras');

  @override
  functionParameters() {
    return '"tipo"'
        ':'
        '$_tipo'
        ','
        '"dados"'
        ':'
        '"$_dados"'
        ','
        '"altura"'
        ':'
        '$_altura'
        ','
        '"largura"'
        ':'
        '$_largura'
        ','
        '"HRI"'
        ':'
        '$_HRI';
  }
}
